guides/customizing-component-styles.md

```diff
@@ -1,73 +1,106 @@
 # Customizing Angular Material component styles
 
-## Styling concepts
+Angular Material supports customizing component styles via Sass API as described in the [theming
+guide][]. This document provides guidance on defining custom CSS rules that directly style
+Angular Material components.
 
-There are 3 questions to keep in mind while customizing the styles of Angular Material
-components:
+[theming guide]: https://material.angular.io/guide/theming
 
-1. Are your styles encapsulated?
-2. Are your styles more specific than the defaults?
-3. Is the component a child of your component, or does it exist elsewhere in the DOM?
+## Targeting custom styles
+
+### Component host elements
+
+For any Angular Material component, you can safely define custom CSS for a component's host element
+that affect the positioning or layout of that component, such as `margin`, `position`, `top`,
+`left`, `transform`, and `z-index`. You should apply such styles upon be defining a custom CSS
+class and applying that class to the component's host element.
+
+Avoid defining custom styles that would affect the size or internal layout of the component, such as
+`padding`, `height`, `width`, or `overflow`. You can specify `display: none` to hide a component,
+but avoid specifying any other `display` value. Overriding these properties can break components
+in unexpected ways as the internal styles change between releases.
+
+### Internal component elements
+
+Avoid any custom styles or overrides on internal elements within a Angular Material components.
+The DOM structure and CSS classes applied for each component may change at any time, causing custom
+styles to break.
+
+## Applying styles to Angular Material components
+
+While Angular Material does not support defining custom styles or CSS overrides on components'
+internal elements, you might choose to do this anyway. There are three points to consider while
+customizing styles for Angular Material components: view encapsulation, CSS specificity, and
+rendering location.
 
 ### View encapsulation
 
-By default, Angular component styles are scoped to affect the component's view. This means that
-the styles you write will affect all the elements in your component template. They will *not*
-affect elements that are children of other components within your template. You can read more
-about view encapsulation in the
+By default, Angular scopes component styles to exclusively affect that component's view. This means
+that the styles you author affect only the elements directly within your component template.
+Encapsulated styles do *not* affect elements that are children of other components within your
+template. You can read more about view encapsulation in the
 [Angular documentation](https://angular.io/guide/component-styles#view-encapsulation). You may
-also wish to take a look at
+also wish to take a review
 [_The State of CSS in Angular_](https://blog.angular.io/the-state-of-css-in-angular-4a52d4bd2700)
```

Comment:
We're probably due for an update on _The State of CSS in Angular_ with these new changes, this article is a bit dated (both date and content).

Reply:
Yeah, we should probably update our guidance once we drop IE11 support and we can throw our weight behind css variables (and maybe constructible stylesheets will be a thing by then, too)